rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -106,6 +106,10 @@ find /srv/salt -maxdepth 1 -type f -iname '*.top' \
     | xargs qubesctl top.enable > /dev/null
 
 %changelog
+* Mon Jun 29 2020 SecureDrop Team <securedrop@freedom.press> - 0.4.0-rc1
+- Consolidates updates from two stages into one
```

Comment:
should we add pr or issue numbers here?

Reply:
good question. so far we haven't in this changelog. since this is a build artifact that can exist independently of the repo that those IDs reference and that will not be linked to from the outside, my inclination is to avoid those references here